apps/namada-interface/src/slices/accounts.ts

```diff
@@ -193,18 +193,55 @@ const balancesAtom = (() => {
 
       set(base, {});
 
-      accounts.forEach(async (account) => {
-        const result = await namada.queryBalances(account.address, [
-          nativeToken || tokenAddress,
-        ]);
-        const balance = result.reduce(
-          (acc, curr) => ({ ...acc, [curr.token]: new BigNumber(curr.amount) }),
-          {} as Balance
-        );
-        set(base, { ...get(base), [account.address]: balance });
+      // Split accounts into transparent and shielded
+      const [transparentAccounts, shieldedAccounts] = accounts.reduce((acc, curr) => {
```

Comment:
How do you create an account from inserting a spending key ? 
It  seems that this is my main issue with adding a spending key, it is not registered in the "accounts'

Reply:
Sure, i''ve pinged you on discord MP !